var/spack/repos/builtin/packages/rose/package.py

```diff
@@ -29,19 +29,19 @@
 from spack import *
 
 
-class Rose(Package):
+class Rose(AutotoolsPackage):
     """A compiler infrastructure to build source-to-source program
        transformation and analysis tools.
        (Developed at Lawrence Livermore National Lab)"""
 
     homepage = "http://rosecompiler.org/"
-    url = "https://github.com/rose-compiler/rose/archive/v0.9.7.tar.gz"
+    #url = "https://github.com/rose-compiler/rose/archive/v0.9.7.tar.gz"
```

Comment:
Hmm, that's a shame. Well I guess we can leave the old version out for now. But it would be really nice to use tarballs in the future. It makes it much easier to distribute software, although Spack has pretty good `git` support.